data/README.md

```diff
@@ -0,0 +1,28 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Apache Arrow Data Experiments
+
+This subdirectory contains experimental Arrow data whose purpose has not
+yet become clear but may be useful in the future. This currently includes
+data used to generate compelling examples that is more realistic than
+generated data or the testing data found in
+[apache/arrow-testing](http://github.com/apache/arrow-testing). This
+subdirectory is intended as a semi-temporary staging area: eventually,
+data here should find a permanent home elsewhere or be removed.
```

Comment:
```suggestion
data here should find a permanent home elsewhere or be removed.

> [!IMPORTANT]  
> Please install and use [Git LFS](https://git-lfs.com) when contributing to this subdirectory. Track large data files with `git lfs track` and add any new large file extensions to [`.gitattributes`](https://github.com/apache/arrow-experiments/blob/main/.gitattributes).
```

Reply:
This is just a notice to other contributors which I think might help future users of this repo!

Reply:
Oh oops I didn't see that you merged already! Thanks